hardware/hardware.mk

```diff
@@ -15,4 +15,7 @@ CACHE_SRC_DIR:=$(CACHE_DIR)/hardware/src
 VSRC+=$(wildcard $(CACHE_HW_DIR)/src/*.v) \
 $(MEM_DIR)/reg_file/iob_reg_file.v \
 $(MEM_DIR)/fifo/afifo/afifo.v \
-$(MEM_DIR)/sp_ram/iob_sp_mem.v
+$(MEM_DIR)/sp_ram_be/iob_sp_ram_be.v \
+$(MEM_DIR)/sp_ram/iob_sp_mem.v \
+$(AXI_MEM_DIR)/rtl/axi_ram.v \
```

Comment:
AXI_MEM is only used for simulation. This file is only for implementation hardware. You should make your own simulation.mk file.

iob_sp_ram_be is not used in the system. It should not be uesed at all because Altera tools do not support it.

Reply:
please fix